src/charactersheet/models/common/treasure.js

```diff
@@ -20,21 +20,26 @@ export function Treasure() {
     self.copper = ko.observable(0);
 
     self.worthInGold = ko.computed(function(){
-        var parsedPlatinum = parseInt(self.platinum()) || 0;
-        var parsedGold = parseInt(self.gold()) || 0;
-        var parsedElectrum = parseInt(self.electrum()) || 0;
-        var parsedSilver = parseInt(self.silver()) || 0;
-        var parsedCopper = parseInt(self.copper()) || 0;
+        const parsedPlatinum = parseInt(self.platinum()) || 0;
```

Comment:
Just curious, do we still need `parseInt` here? The inputs are numbers, so that should restrict non-numeric characters. Or did we find that not all browsers enforce that or something?

Reply:
Not all browsers enforce that unfortunately...